x-pack/plugins/security_solution/public/detections/pages/detection_engine/rules/types.ts

```diff
@@ -190,7 +193,7 @@ export interface ScheduleStepRule {
 }
 
 export interface ActionsStepRule {
-  actions: AlertingRuleAction[];
+  actions: AlertingRuleAction[] | AlertingRuleSystemAction[];
```

Comment:
Shouldn't this be `(AlertingRuleAction | AlertingRuleSystemAction)[]` ?

My understanding was that the `actions` array was a mixture of both types of actions.